src/main/java/com/patternpedia/api/PatternPediaAPI.java

```diff
@@ -16,6 +20,7 @@
 @Slf4j
 @RestController
 @SpringBootApplication
+@OpenAPIDefinition(info = @Info(title = "pattern-atlas-api", version = "1.0", contact = @Contact(url = "https://github.com/PatternAtlas/pattern-atlas-api", name = "GitHub Repository")))
 public class PatternPediaAPI implements CommandLineRunner {
```

Comment:
can we set name to something more meaningful than "Github Repository"?

Reply:
Done